examples/output_voltage_measurement/README.md

```diff
@@ -18,10 +21,6 @@ commands using NI-VISA.
     sections of the main sequence. For **Test UUTs** and batch process model use
     cases, these steps should be moved to the `ProcessSetup` and
     `ProcessCleanup` callbacks.
-- Pin-aware, supporting one session and pin per instrument and a single site
-  - Sources the DC voltage level of the DUT input pin
-  - Measures the voltage of the DUT output pin
-- Includes InstrumentStudio and MeasurementLink UI Editor project files
```

Comment:
Removed this as it's a duplicate of [line 15](https://github.com/ni/measurementlink-python/pull/665/files#diff-a6864a799339e1795f8edf148ed06c93342a50cbba30a220aceacf1a10fed590R15).